src/enums.ts

```diff
@@ -37,7 +37,7 @@ export class Messages {
 
 
 export class ThemeTypes {
-    public static Both = 'both';
-    public static Ligth = 'vs';
-    public static Dark = 'vs-dark';
+    public static Both: { key: string, value: string } = { key: 'Both', value: 'both', };
+    public static Ligth: { key: string, value: string } = { key: 'Ligth', value: 'vs', };
+    public static Dark: { key: string, value: string } = { key: 'Dark', value: 'vs-dark', };
```

Comment:
Why this change? You are using it in a way that I don't immediately understand.

Reply:
Oh yes: when i added it i was modelling it as a key-value pair, but the right terms would be `display_member` and `value_member`.
This will improve the user experience, 
because if an user reads:
"Both"
"Ligth"
"Dark"
can definitley better understand the question, instead of reading: (user reactions in parens)
"both" (?)
"vs" (?????)
"vs-dark" (ohhh, now i think i can understand what the question may refers to).

I'm not sure if a better code structure is available;
Anyway, I remove the last trailing commas 👍🏼

Reply:
Now I get it, it,s the similar concept of a dropdown menù, the user see a pretty string, for the program internals the id is better.